### PR TITLE
enhance(main/cmus): enable modplug and wavpack input plugins

### PR DESCRIPTION
**Pull request by pgaskin**
The dependencies weren't around when the cmus package was added.

### CODE REVIEW

**Review by twaik — APPROVED** [review]
LGTM. @truboxl, Probably you have objection about the order of `TERMUX_PKG_DEPENDS` items?

File: packages/cmus/build.sh
Comment (truboxl) [inline packages/cmus/build.sh:7]:
```suggestion
TERMUX_PKG_DEPENDS="ffmpeg, libandroid-support, libflac, libiconv, libmad, libmodplug, libvorbis, libwavpack, ncurses, opusfile, pulseaudio"
```

File: packages/cmus/build.sh
Comment (truboxl) [inline packages/cmus/build.sh:8]:
```suggestion
TERMUX_PKG_SRCURL=https://github.com/cmus/cmus/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```

**Comment by twaik** [comment]
I will merge it in 24 hours if no objections or after it gets 2+ approves.

**Comment by pgaskin** [comment]
bump

**Comment by twaik** [comment]
Oh, right.

**Pull request merged**
